api/api.go

```diff
@@ -315,3 +315,33 @@ type ArchiveOpenOptions struct {
 type ArchiveCreateOptions struct {
 	LogSizeThreshold *int64 `json:"log_size_threshold,omitempty"`
 }
+
+// AuthAPIAudience is the audience claim for any zqd api access token.
+// Though represented as a URL, it does not need to be a reachable location.
+const AuthAPIAudience = "https://app.brimsecurity.com"
```

Comment:
Out of curiosity, do imagine this will always be hardcoded thusly? Would we want this to be in ppl?

Reply:
( @nwt asked a similar question below). The audience needs to be known to the client, as it needs to use it when it asks for an access token from Auth0. I hadn't put that in the method response as it doesn't need to be different between different environments, but I think it'd be more consistent to do so.